public/js/actions/tests/loadSourceText.js

```diff
@@ -74,8 +75,8 @@ describe("loadSourceText", () => {
     const store = createStore(simpleMockThreadClient);
 
     Task.spawn(function* () {
-      yield store.dispatch(loadSourceText({ id: "foo1" }));
-      yield store.dispatch(loadSourceText({ id: "foo1" }));
+      yield store.dispatch(loadSourceText(fromJS({ id: "foo1" })));
+      yield store.dispatch(loadSourceText(fromJS({ id: "foo1" })));
```

Comment:
This is why actions always just receive normal JS objects :) We want to optimize for the common use case, and with actions you are always passing things in like this.